flixel/animation/FlxAnimationController.hx

```diff
@@ -777,6 +777,55 @@ class FlxAnimationController implements IFlxDestroyable
 		play(AnimName);
 		return AnimName;
 	}
+		
+	public function animationList():Array<FlxAnimation>
+	{
+		var arrayNames:Array<FlxAnimation> = [];
+		
+		for (i in _animations)
+		{
+			arrayNames.push(i);
+		}
+
+		return arrayNames;
+	}
+
+	public function nameList():Array<String>
+	{
+		var anims = animationList();
+		var names:Array<String> = [];
+		for (i in 0...anims.length)
+		{
+			names.push(anims[i].name);
+		}
+
+		return names;
+	}
+
+	public function exists(name:String):Bool
+	{
+		return nameList().contains(name);
+	}
+	
+	public function renameAnimation(name:String, value:String)
```

Comment:
can just be called `rename`

needs documentation

for clarity, use `oldName` and `newName`

Reply:
Yeah, changed the names of the vars and the function itself, also added some docs.
Resolved I guess